tox.ini

```
@@ -105,3 +106,10 @@ extras = docs
 commands =
     pip freeze
     sphinx-build -W -b linkcheck . _build/html
+
+[testenv:codestyle]
+skip_install = true
+changedir = {toxinidir}
```

want to run this in the tox env, not the local working dir
```suggestion
changedir = .
```